### ISSUE TITLE
I see scheduler.step() is being invoked right before the for loop in the train function or before training for one epoch

### ISSUE THREAD

**Issue by Phuoc-Hoan-Le** [liuzechun/ReActNet#32]
In line 185, for the mobilenet in directory, 1_step1, I see scheduler.step() is being invoked right before the for loop in the train function. Shouldn't it be right after the for loop is finished?

**Comment by Phuoc-Hoan-Le** [comment]
@liuzechun @szq0214 Any update on this?
